docs/csharp/delegates-events.md

```diff
@@ -41,3 +41,4 @@ This topic will be covered under the following articles:
 
     This article discusses how you should distinguish between using events and delegates in your designs.
  
+You can download the samples code for the [delegates sample](https://github.com/dotnet/samples/tree/master/csharp/delegates-and-events) and the [events sample](https://github.com/dotnet/samples/tree/master/csharp/events) from out GitHub samples repository.
```

Comment:
download the samples code --> download the sample code
but perhaps a more concise version: "download the \[delegates sample](…)